SequencePhenotype.java

```diff
@@ -88,8 +88,15 @@ public double riskOfInfection(Phenotype[] history) {
 
     // returns a mutated copy, original SequencePhenotype is unharmed
     public Phenotype mutate() {
-        int index = random();
-        return new SequencePhenotype(this.sequence + NUCLEOBASES[index]);
+        Random rand = new Random();
+        int index = rand.nextInt(0, this.sequence.length() - 1);
+        int indexN = rand.nextInt(0, this.NUCLEOBASES.length - 1);
```

Comment:
This is a nitpick -- but perhaps change these variable names to be a bit more descriptive (i.e., `int indexSite` and `int indexNucleotide`?

Reply:
Great catch! I'll update it.